audio/engine.asm

```diff
@@ -212,7 +212,7 @@ _UpdateSound::
 UpdateChannels:
 	ld hl, .ChannelFnPtrs
 	ld a, [wCurChannel]
-	and $7
+	maskbits NUM_MUSIC_CHANS ; bit 0-2 (current channel)
```

Comment:
Since `NUM_MUSIC_CHANS` has a meaningful name, the comment is no longer necessary to explain the meaning of the `$7` value; likewise for the other line.